root_finder.py

```diff
@@ -19,11 +19,11 @@
 # When run, the code should print the iteration number, the new value
 # of x, x_new, and the value of f(x_new) at each iteration.
 
-  def f(x):
+def f(x):
   # evaluate f at x
   return x**3 + 4*x**2 - 10
 
-def fprime(x)
+def fprime(x):
   # evaluate f' at x
     return 3*x + 8*x
```

Comment:
you might want to look more closely at whether this function is doing the right thing...